1.14/test/run

```diff
@@ -144,6 +146,31 @@ test_command() {
   return 0
 }
 
+test_logs() {
+  local expected="$1"
+  local message="$2"
+  local logfile="$3"
+
+  if [ "$message" ]; then
+    info ${message}
+  fi
+  if [ -n "$logfile" ]; then
+      out=$(docker cp $(cat ${cid_file}):${logfile} - | tar -xvO '*')
+  elif [ "$logfile" = "stdout" ]; then
+      out=$(docker logs $(cat ${cid_file}) 2>/dev/null)
+  elif [ "$logfile" = "stderr" ]; then
+      out=$(docker logs $(cat ${cid_file}) 1>/dev/null 2>&1)
```

Comment:
I think the problem here is `1>/dev/null 2>&1`. Wouldn't that redirect all output to `/dev/null`?